sfge/src/main/java/com/salesforce/graph/symbols/apex/schema/DescribeSObjectResult.java

```diff
@@ -120,6 +119,15 @@ public Optional<ApexValue<?>> executeMethod(
                         .methodVertex(method);
         String methodName = method.getName();
 
+        Optional<ApexValue<?>> optApexValue = _applyMethod(invocableExpression, builder, methodName);
+
+        if (!optApexValue.isPresent()) {
+            optApexValue = Optional.of(ApexValueUtil.synthesizeReturnedValue(builder, method));
+        }
+        return optApexValue;
+    }
+
+    private Optional<ApexValue<?>> _applyMethod(InvocableWithParametersVertex invocableExpression, ApexValueBuilder builder, String methodName) {
```

Comment:
Similar change on all the schema values. I've restructured so that method invocations that we know about don't get missed from for-loop invocation, which uses `apply` method.